### ISSUE TITLE
Error alerts in wishlist

### ISSUE THREAD

**Issue by carinadues** [scandipwa/scandipwa#3282]
**Description**:
When you try to add a product to the cart from the wishlist and it has required options, there are two big red erros.

**To Reproduce**:
1. Open https://scandipwapmrev.indvp.com/default/28-virtual-product.html
2. Login
2. Add the product to your wishlist
3. Go to the wishlist
4. Tap Add to cart. There will be two errors.
5. You can also try with budles: https://scandipwapmrev.indvp.com/default/bundle-sandals-with-fixed-prices.html

**Actual result**:
![image](https://user-images.githubusercontent.com/82165392/132051730-dd3f0980-5a1f-4c27-9707-d51e14071d8a.png)

**Expected behavior**:
Here there are these options:
1 - make the first alert yellow instead of red and remove the second alert.
2 - instead of showing errors, direct the user to the product page.
3 - both of the above.

**Comment by tatiana-scandi** [comment]
Put on hold till merge of https://github.com/scandipwa/scandipwa/pull/3236 (as suggested by @carinadues ).

Errors above are a concequence of selected customisable options not being sent to the server in add to wishlist request.
E.g. for product https://scandipwapmrev.indvp.com/default/28-virtual-product.html, if you select both required customisable options, `saveWishlistItem` request will look like this:
```
{
  "query": "mutation($wishlistItem_1:WishlistItemInput!){saveWishlistItem:s_saveWishlistItem(wishlistItem:$wishlistItem_1){id,sku,qty,description,price,price_without_tax,buy_request,options{label,value}}}",
  "variables": {
    "wishlistItem_1": {
      "sku": "Virtual Hat",
      "quantity": 1
    }
  }
}
```

While variables should include info on what options were selected. Something like:
```
{
"variables": {
    "wishlistItem_1": {
        "sku": "Bundle Sandals with fixed prices",
        "product_option": {
          "extension_attributes": {
            "customisable_options": [
              {
                "id": 70,
                "value":  "159"
              },
              {
                "id": 69,
                "value": "151"
              }
            ]
          }
      }
```

As a result `info_buyRequest` is not saved in DB correctly (`wishlist_item_option` table) and there is no way for add to cart from wishlist to work after that.

**Comment by tatiana-scandi** [comment]
Now blocked by https://github.com/scandipwa/scandipwa/issues/3134 (hashed ids are being sent to the server now, but it is impossible to check if fetching wishlist and consequent adding to cart works)

**Comment by Elena-Evsa** [comment]
Now only one error message is appear
About color - ok
We have the following logic
if error message comes from server - it is red color
if error message validated on client side - it has yellow color
